packages/angular/src/generators/ngrx/ngrx.classes.spec.ts

```diff
@@ -0,0 +1,278 @@
+import { Tree } from '@nrwl/devkit';
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import { dirname } from 'path';
+import {
+  AppConfig,
+  createApp,
+  createLib,
+  getAppConfig,
+  getLibConfig,
+} from '../../utils/nx-devkit/testing';
+import { ngrxGenerator } from './ngrx';
+import { NgRxGeneratorOptions } from './schema';
+
+describe('NgRx generator', () => {
+  let appConfig: AppConfig;
+  let statePath: string;
+  let tree: Tree;
+
+  const defaultOptions: NgRxGeneratorOptions = {
+    directory: '+state',
+    minimal: true,
+    module: 'apps/myapp/src/app/app.module.ts',
+    name: 'users',
+    useDataPersistence: false,
+    syntax: 'classes',
+  };
+
+  const expectFileToExist = (file: string) =>
```

Comment:
Yeah, it could, I thought about doing so, but I was undecided on whether that would be really something we want to expose as part of the devkit or not. There could be several similar small convenience utilities we could have (e.g. `expectFileInTreeToContain`, `expectFileInTreeToMatchSnapshot` and more), but I'm not sure if these small wrappers are something we want to maintain or not.

Probably better to check with Jason first. I'd keep it in a separate PR if it's decided to add them.

Reply:
Yeah, a new PR for them would be the correct route I believe. No worries